src/picasso_global.h

```diff
@@ -11,12 +11,22 @@
 #include "math_type.h"
 #include "data_vector.h"
 #include "fixedopt.h"
+#include <string.h>
 
 #include "picasso.h"
 
+inline void* ZeroBufferAlloc(size_t bytes) {
+	void *buffer = mem_malloc(bytes);
+    return buffer?memset(buffer, 0, bytes):NULL;
+}
+inline void* ZeroBuffersAlloc(size_t num, size_t size) {
+	void *buffer = mem_calloc(num, size);
```

Comment:
mem_calloc  will call  calloc  is zeroed

Reply:
@onecoolx Thank you for telling me this.:smile: